common/views/themes/default.js

```diff
@@ -1,6 +1,19 @@
 // @flow
 import { css, keyframes } from 'styled-components';
+import { type SpaceOverrides } from '../components/styled/Space';
 
+const spacingUnits = {
+  '1': 4,
```

Comment:
Is there a reason we're not using `number` type keys?

Reply:
"non-string literal property keys are not supported. Flow (inferError)"

Looks like there's been [beef with this since 2015](https://github.com/facebook/flow/issues/380).

Options look like:

1. use a string
2. put it in an array
3. use a map

I'm happy enough with what we've got working as it is now, but equally happy to implement array/map if you think there's a real benefit?

Reply:
This is good, just curious.

Reply:
This is good, just curious.